cmd/applyconfig/applyconfig.go

```diff
@@ -139,8 +140,20 @@ func (c *configApplier) asGenericManifest() error {
 	return err
 }
 
-func (c configApplier) asTemplate() error {
-	ocProcessCmd := makeOcCommand(ocProcess, c.path, c.user)
+func (c configApplier) asTemplate(params []templateapi.Parameter) error {
```

Comment:
/hold

need to check no leaking in the output of the env. vars which are secret in the log.

Reply:
/hold cancel

risk is limited as discussed with @petr-muller 

I will work on the logger with censor